cosmoz-tree.js

```diff
@@ -78,11 +85,13 @@ export class Tree {
 	 * @returns {Array|Object|String} The value of the property
 	 * @param {node} node (The node of which the property value should be returned of.)
 	 * @param {String} propertyName (The name of the property. e.g. "name")
-	*/
-	getProperty(node, propertyName) { // eslint-disable-line no-unused-vars
+	 */
+	getProperty(node, propertyName) {
 		throw new Error('Must be implemented in derived object');
 	}
 
+	/* eslint-enable no-unused-vars */
```

Comment:
forgotten comment?

Reply:
No. I first disable and then re-enable the rule.